CONTRIBUTING.md

```diff
@@ -120,6 +120,25 @@ docker run --rm --runtime=my-branch --rm hello-world
 make refresh
 ```
 
+### Update golang dependencies
+
+First, we need to update dependencies in the go.mod and go.sum files. To do
+that, we should checkout the go branch and update dependencies using the go get
+tool.
+
+```bash
+git checkout origin/go
+go get golang.org/x/net
+git checkout origin/master
```

Comment:
nit: move this to the next bash block, since the next sentence talks about "we should checkout the master branch and..."